### ISSUE TITLE
Replace old screenshot with new one 

### ISSUE THREAD

**Issue by parispittman** [swiftlang/swift#75109]
### Description

Doc to change: https://github.com/swiftlang/swift/blob/main/docs/HowToGuides/FirstPullRequest.md

The screenshot in this doc is showing apple/swift and it's now swiftlang/swift under submitting a pull request. Good first issue for someone! 
[Link to image](https://github.com/swiftlang/swift/blob/main/docs/GitHubCreatePRScreenshot.png)

### Reproduction

n/a

### Expected behavior

n/a

### Environment

n/a

**Comment by vidhupv** [comment]
Hi, I would like to start contributing and work on this as my first issue. The current screenshot has a push ready to be pulled( which I assume has to be there considering its in the section for pull requests) but this feels like a paradox considering I can only send in my first pull request AFTER taking the screenshot. I would appreciate any tips on how to move forward!

**Comment by MadGanGithub** [comment]
Hey @parispittman  
I would like to work on this issue
Can you assign me?

**Comment by abhinav26966** [comment]
Hi, I'm interested in working on this issue and would love to contribute to the project. Could you please assign it to me? @parispittman

**Comment by AnthonyLatsis** [comment]
@vidhupv was first.

**Comment by AnthonyLatsis** [comment]
> Should I just push a dummy change to take the screenshot?

Yeah, you just need to push a dummy branch to your fork for that button to show up.

**Comment by wesleykebrown** [comment]
Hi, @parispittman. I'm working on my first open source contribution for my software development cohort and I would like to be added to this issue ticket, please.

**Comment by AnthonyLatsis** [comment]
Hello Wesley, this issue is already being worked on. See: [How do I pick something to work on?](https://github.com/swiftlang/swift/blob/main/docs/HowToGuides/FirstPullRequest.md#how-do-i-pick-something-to-work-on). You could try https://github.com/swiftlang/swift/issues/57499 if you are looking to contribute code as well.
